### DISCUSSION TITLE
Unable to add WebAnalytics to my Astro website

### DISCUSSION THREAD

**Discussion by alns0dev** [vercel/community#4435] [Help] [answered]
### Summary

I have an Astro website currently working just fine, and as I'm going to start a blog on it, I went on to follow the WebAnalytics guide from Vercel and, even though I did everything as it stated (which were only 3 steps), it fails.
My website had the output set to `static` on the astro.congif.mjs before following Vercel's guide,

### Steps to Reproduce

Running `yarn build` give me the following error: `Package subpath './errors' is not defined by "exports" in /home/user/company/website/node_modules/astro/package.json`

To reproduce it locally:

1. Create an Astro domain with Vite PWA.
2. Add WebAnalytics to your astro.config.mjs with the import from @astrojs/vercel/serverless
3. Run `yarn build`, `npm run build` or `pnpm run build`

**Comment by alns0dev** [comment, marked as answer]
I solved this by changing the output and the import to static instead of server. Not that documentation helped.

**Reply by florian-lefebvre** [reply]
I don't know if you reached out on the Astro's Discord but there were several people talking about that issue

**Reply by alns0dev** [reply]
Hey @florian-lefebvre! Thanks for reaching out. Yes, I was part of the people talking about it and this was the solution.
